src/models.zig

```diff
@@ -60,11 +60,12 @@ const Quad = struct {
 	uvs: [4]usize,
 };
 
-pub const ModelIndex = packed struct {
-	index: u32,
+pub const ModelIndex = enum(u32) {
+	no_value = std.math.maxInt(u32),
```

Comment:
Why?

Reply:
let's keep the discussion in issue comments instead of code comments under individual cases.